cmd/cloud/organization_test.go

```diff
@@ -73,4 +74,14 @@ func TestOrganizationExportAuditLogs(t *testing.T) {
 		_, err := execOrganizationCmd(cmdArgs...)
 		assert.NoError(t, err)
 	})
+
+	// Delete audit logs exports
+	currentDir, _ := os.Getwd()
```

Comment:
these test cases where leaving exported files. just patching those test cases to delete those exported files

Reply:
yes I noticed them too. I was wondering why they were showing up when `execOrganizationCmd` already sets its output to go to a buffer as opposed to `os.stdout`